engine/etcdng/etcd.go

```diff
@@ -32,14 +35,17 @@ type Options struct {
 	Box             *secret.Box
 }
 
-func New(nodes []string, etcdKey string, registry *plugin.Registry, options Options) (engine.Engine, error) {
+func New(nodes []string, etcdKey, etcdCaFile, etcdCertFile, etcdKeyFile string, registry *plugin.Registry, options Options) (engine.Engine, error) {
```

Comment:
Could you please move these parameters to Options instead? this will simplify the constructor